data-prepper-api/src/main/java/com/amazon/dataprepper/model/configuration/PipelineModel.java

```diff
@@ -67,9 +67,11 @@ else if (preppers != null) {
     @JsonProperty("sink")
     private final List<PluginModel> sinks;
 
+    @JsonProperty("workers")
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private final Integer workers;
 
+    @JsonProperty("delay")
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private final Integer readBatchDelay;
```

Comment:
Would it be better to just rename this variable `delay` as well?

Reply:
This change is simpler. I imagine the reasoning to name it `readBatchDelay` was to be more descriptive